src/main.rs

```diff
@@ -523,6 +529,11 @@ fn run(args: Cli) -> Result<i32> {
                     if history_print.response_headers {
                         printer.print_response_headers(prev_response)?;
                     }
+                    if args.stream.is_none()
+                        && is_text_event_stream_body(prev_response) == Some(true)
```

Comment:
Can you do this check inside `Printer::print_response_body()`? We are already extracting the response's `content_type` on L448 and use it to determine how to format the response body.

Reply:
Sure